app/models/match.rb

```diff
@@ -1,3 +1,17 @@
 class Match < ApplicationRecord
   belongs_to :field
+  
+  def self.update_match_data(req_id)
+    m_request = MatchRequest.find_by_id(req_id)
+    m = Match.find_by_id(m_request.match_id)
+    if m_request.status == "ACCEPT"
+      m.team_away_id = m_request.team_request_id
```

Comment:
Please change this line
`m.team_away_id = m_request.team_request_id` to `m.team_away_id = m_request.requested_team_id`